build.assets/Dockerfile

```diff
@@ -159,6 +159,15 @@ RUN apt-get -y update && \
         xauth \
         libpcsclite-dev \
         && \
+    sudo install -m 0755 -d /etc/apt/keyrings && \
```

Comment:
sudo does not appear to be needed - all the other apt commands run without it, so presumably this container runs as root.